portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -47,10 +82,49 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
         //response[1] = capitals.get(4);
         Random rand = new Random();
 
-        response.setContentType("text/html;");
-        response.getWriter().println(countries.get(rand.nextInt(countries.size())));
+        // response.setContentType("text/html;");
+        // response.getWriter().println(countries.get(rand.nextInt(countries.size())));
+        String json3 = "{";
+        json3 += "\"gender\": ";
+        json3 += "\"" + "Male" + "\"";
+        json3 += ", ";
+        json3 += "\"height\": ";
+        json3 += "\"" + "1.75" + "\"";
+        json3 += "}";
+
+        String json = convertArrayListToJson(hardCodedJson);
+        // String json = convertToJsonUsingGson(hardCodedJson);
+        response.setContentType("application/json;");
+        response.getWriter().println(json);
     }
 
+    public String convertArrayListToJson(List<String> list) {
+        // Starting json string
+        String json = "{";
+        json += "\"messages\": [";
+        
+        // Adding all hard-coded messages
+        for (int i = 0; i < list.size()-1; i++) {
+            json += "\"" + list.get(i) + "\",";
```

Comment:
Note: When the comments are user-submitted, this is no longer "safe". Comments that include double-quotes and other JSON syntax elements will break the encoding.

GSon and other libraries will escape strings correctly.